src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue27846.cs

```diff
@@ -0,0 +1,26 @@
+#if IOS
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue27846 : _IssuesUITest
+{
+	public Issue27846(TestDevice testDevice) : base(testDevice)
+	{
+	}
+
+	public override string Issue => "[iOS] More tab doesn't respect shell nav bar customization";
+
+	[Test]
+	[Category(UITestCategories.Shell)]
+	[Category(UITestCategories.TabbedPage)]
+	public void MoreTabShouldRespectNavBarCustomization()
+	{
+		App.WaitForElement("More");
+		App.Click("More");
+		VerifyScreenshot();
```

Comment:
Pending snapshots, already available in the latest build.

Example:
![image](https://github.com/user-attachments/assets/b1410f95-c133-4ee7-ae85-6fdddf61e06a)

Could you commit the images?